docs/resources/branch.md

```diff
@@ -0,0 +1,62 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "gitlab_branch Resource - terraform-provider-gitlab"
+subcategory: ""
+description: |-
+  This resource allows you to create and manage branch https://docs.gitlab.com/ee/user/project/repository/branches/ for your GitLab projects.
+---
+
+# gitlab_branch (Resource)
+
+This resource allows you to create and manage [branch](https://docs.gitlab.com/ee/user/project/repository/branches/) for your GitLab projects.
+
+## Example Usage
+
+```terraform
+resource "gitlab_group" "this" {
```

Comment:
🤔  did you add this manually to the `branch.md`? It might be overridden when regenerating the docs, right?

Reply:
Sorry, I skip this step in CONTRIBUTING.md.